File: pi/pi.java
public class pi {
    public static void main(String[] args) {

        int n = 10000000;
        int mc = 0;

        for (int i = 0; i < n; i++) {
            double x = Math.random();
            double y = Math.random();

            if (x * x + y * y <= 1) {
                mc++;
            }

        }

        double pi = (double)4*mc/n;

        System.out.println("pi="+pi);
        
    }
}

File: pi/pi.js
var n = 10000000
var mc = 0

for(var i = 0; i < n; i++) {

    var x = Math.random()
    var y = Math.random()

    if(x**2 + y**2 < 1) {
        mc++;
    }
    
}

var pi = 4 * mc / n;

console.log(pi);